mix.exs

```diff
@@ -18,6 +18,8 @@ defmodule Mana.MixProject do
       start_permanent: Mix.env() == :prod,
       dialyzer: [
         ignore_warnings: ".dialyzer.ignore-warnings",
+        plt_add_apps: [:mix],
+        plt_add_deps: :transitive,
```

Comment:
This could be removed. My understanding is that it's ensuring we include types from dependencies.